src/main.rs

```diff
@@ -159,7 +159,7 @@ fn main() {
     for argument in argvs.iter_mut() {
         argument.zeroize();
     }
-    // Only one request every 1500 milliseconds from any given IP
+    // Only one request every 1600 milliseconds from any given IP
```

Comment:
HIBP limits at 1500 milliseconds. Adding another 100 was just to give it some extra room, but maybe the comment should mention that 1500 is the official limit instead? If so, we need to change it in `main.rs` as well.

Reply:
Done. Here's what I added into the code:

```
// HIBP limits requests to one per 1500 milliseconds. We're allowing for 1600 below as a buffer.
```